Transform failing with `java.lang.NoClassDefFoundError` on Windows

### Checked for duplicates

Yes - I've already checked

### 🐛 Describe the bug

When I did transform on windows with Java jdk 21 on a Windows machine, I noticed it was throwing a `java.lang.NoClassDefFoundError` exception

### 🕵️ Expected behavior

I expected transformation to execute successfully

### 📜 To Reproduce

```
c:\Program Files\transform-1.12.2\bin>transform -O c:\Users\user\Documents\transform\GRV_JUGR_2022055_2005X35MCMMMV01.xml
Exception in thread "main" java.lang.NoClassDefFoundError: com/sun/xml/bind/v2/model/annotation/AnnotationReader
        at java.base/java.lang.ClassLoader.defineClass1(Native Method)
        at java.base/java.lang.ClassLoader.defineClass(ClassLoader.java:1027)
        at java.base/java.security.SecureClassLoader.defineClass(SecureClassLoader.java:150)
        at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(BuiltinClassLoader.java:862)
        at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(BuiltinClassLoader.java:760)
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(BuiltinClassLoader.java:681)
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:639)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
        at java.base/java.lang.Class.getDeclaredMethods0(Native Method)
        at java.base/java.lang.Class.privateGetDeclaredMethods(Class.java:3578)
        at java.base/java.lang.Class.getMethodsRecursive(Class.java:3719)
        at java.base/java.lang.Class.getMethod0(Class.java:3705)
        at java.base/java.lang.Class.getMethod(Class.java:2393)
        at javax.xml.bind.ContextFinder.newInstance(ContextFinder.java:205)
        at javax.xml.bind.ContextFinder.newInstance(ContextFinder.java:166)
        at javax.xml.bind.ContextFinder.find(ContextFinder.java:344)
        at javax.xml.bind.JAXBContext.newInstance(JAXBContext.java:508)
        at javax.xml.bind.JAXBContext.newInstance(JAXBContext.java:465)
        at javax.xml.bind.JAXBContext.newInstance(JAXBContext.java:366)
        at gov.nasa.pds.objectAccess.ObjectAccess.getJAXBContext(ObjectAccess.java:183)
        at gov.nasa.pds.objectAccess.ObjectAccess.getProduct(ObjectAccess.java:199)
        at gov.nasa.pds.objectAccess.ObjectAccess.getProduct(ObjectAccess.java:189)
        at gov.nasa.pds.transform.util.Utility.getFileAreas(Utility.java:171)
        at gov.nasa.pds.transform.util.ObjectsReport.list(ObjectsReport.java:138)
        at gov.nasa.pds.transform.TransformLauncher.processMain(TransformLauncher.java:457)
        at gov.nasa.pds.transform.TransformLauncher.main(TransformLauncher.java:475)
Caused by: java.lang.ClassNotFoundException: com.sun.xml.bind.v2.model.annotation.AnnotationReader
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
        ... 27 more
 
c:\Program Files\transform-1.12.2\bin>echo %java_home%
C:\Program Files\Java\jdk-21.0.2
```

### 🖥 Environment Info

Windows
openjdk 21

### 📚 Version of Software Used

1.12.2

### 🩺 Test Data / Additional context

_No response_

### 🦄 Related requirements

_No response_

### ⚙️ Engineering Details

Pretty certain this is a Java issue. Need to try to replicate and down the issue.

--- a/pom.xml
+++ b/pom.xml
@@ -109,6 +109,7 @@ POSSIBILITY OF SUCH DAMAGE.
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
+        <version>3.3.0</version>
         <configuration>
           <archive>
             <manifest>
@@ -122,6 +123,7 @@ POSSIBILITY OF SUCH DAMAGE.
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>buildnumber-maven-plugin</artifactId>
+        <version>3.1.0</version>
         <executions>
           <execution>
             <phase>validate</phase>
@@ -142,6 +144,7 @@ POSSIBILITY OF SUCH DAMAGE.
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-assembly-plugin</artifactId>
+        <version>3.7.1</version>
         <executions>
           <execution>
             <id>bin-release</id>
@@ -251,9 +254,9 @@ POSSIBILITY OF SUCH DAMAGE.
       <version>2.3.9</version>
     </dependency>
     <dependency>
-      <groupId>javax.xml.bind</groupId>
-      <artifactId>jaxb-api</artifactId>
-      <version>2.3.1</version>
+      <groupId>xalan</groupId>
+      <artifactId>serializer</artifactId>
+      <version>2.7.3</version>
     </dependency>
     <!-- See the JBoss repo for jai-core and jai-codec -->
     <dependency>

